system/iptables.py

```diff
@@ -296,12 +315,13 @@ def main():
         state=module.params['state'],
     )
     ip_version = module.params['ip_version']
+    insert     = module.params['insert']
```

Comment:
PEP 8 don't like the multiple spaces, personally I would have removed them and also put this line above the previous one...